contracts/src/bridge/Bridge.sol

```diff
@@ -104,6 +112,14 @@ contract Bridge is Initializable, DelegateCallAware, IBridge {
             bytes32 acc
         )
     {
+        if (
+            sequencerReportedSubMessageCount != prevMessageCount &&
+            prevMessageCount != 0 &&
+            sequencerReportedSubMessageCount != 0
+        ) {
+            revert BadSequencerMessageNumber(sequencerReportedSubMessageCount, prevMessageCount);
+        }
```

Comment:
should we add a magic value that allows us to skip this check? that means that we could prob remove the old function entrypoint in the release after this one